group_vars/location_chris/general.yml

```diff
@@ -0,0 +1,3 @@
+---
+contact_nickname: 'FETF – Freifunk Engenieering Task Force'
```

Comment:
I'd prefer to also set that contact nickname by default, if thats a community site.